packages/darklang/stdlib/string.dark

```diff
@@ -190,9 +190,10 @@ module Darklang =
 
       /// Checks if <param lookingIn> contains <param searchingFor>
       let contains (lookingIn: String) (searchingFor: String) : Bool =
-        PACKAGE.Darklang.Stdlib.Option.isSome (
-          Builtin.String.indexOf lookingIn searchingFor
-        )
+        if (Builtin.String.indexOf lookingIn searchingFor) == -1 then
```

Comment:
No need for the `if`. You can just do `Builtin.String.indexOf lookingIn searchingFor) != -1`